hw/mcu/sifive/fe310/src/hal_timer.c

```diff
@@ -169,9 +182,9 @@ int
 hal_timer_init(int timer_num, void *cfg)
 {
     struct fe310_hal_tmr *tmr;
+    (void)cfg;
 
-    if (timer_num >= FE310_HAL_TIMER_MAX || !(tmr = fe310_tmr_devs[timer_num]) ||
-        (cfg == NULL)) {
+    if (timer_num >= FE310_HAL_TIMER_MAX || !(tmr = fe310_tmr_devs[timer_num])) {
```

Comment:
Why do you need to set `tmr` if it's not being used?

Reply:
well this change did not added this line, it just removed part of the condition that was totally useless.

...but I will remove it.